portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +38,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String comment = request.getParameter("comment-input");
     commentsList.add(comment);
 
+    Entity commentEntity = new Entity("stringComment");
+    commentEntity.setProperty("comment", comment);
```

Comment:
nit - looks like the guide is pushing for you to store the timestamp as well (which will help with loading in the next PR).